src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -585,9 +599,10 @@ def update_file(ocsp) -> None:
             1. file for OCSP_RESPONSE_VALIDATION_CACHE which keeps ocsp response validation result
             2. ocsp_response_cache.json, the file in the same format as the one downloaded from snowflake cache service
         """
+        ocsp_response_validation_cache = _get_ocsp_response_validation_cache()
         if OCSPCache.CACHE_UPDATED:
-            if isinstance(OCSP_RESPONSE_VALIDATION_CACHE, SFDictFileCache):
```

Comment:
The rest of this file should not change. You should be operating on the global cache every time. We resolve it once and then keep using that one!

Reply:
see comment above, change reverted